client/src/modules/Teams/components/TeamModal/TeamModal.tsx

```diff
@@ -122,8 +117,19 @@ export default function TeamModal({ onCancel, onSubmit, data, courseId, isManage
               },
             ]}
             label="Students"
+            initialValue={mode === 'edit' ? data?.students?.map(s => s.id) : undefined}
           >
-            <StudentSearch onChange={handleChangeStudents as any} keyField="id" courseId={courseId} mode="multiple" />
+            <StudentSearch
+              onChange={handleChangeStudents as any}
+              keyField="id"
+              courseId={courseId}
+              mode="multiple"
+              defaultValues={data?.students?.map(s => ({
+                name: s.fullName,
+                id: s.id,
+                githubId: s.githubId,
+              }))}
```

Comment:
To be honest, I don't like these one letter abbreviations, let's replace it with something more meaningful  
```suggestion
              defaultValues={data?.students?.map(student => ({
                name: student.fullName,
                id: student.id,
                githubId: student.githubId,
              }))}
```